src/Spd.Presentation.Licensing/Controllers/LicenceAppController.cs

```diff
@@ -49,5 +49,20 @@ public async Task<IEnumerable<LicenceAppListResponse>> GetBizLicenceApplications
         {
             return await _mediator.Send(new GetBizLicAppListQuery(bizId), ct);
         }
+
+        /// <summary>
+        /// Cancel Draft Application
+        /// </summary>
+        /// <param name="appId"></param>
+        /// <param name="ct"></param>
+        /// <returns></returns>
+        [Route("api/applications/{appId}")]
+        [HttpDelete]
+        [Authorize(Policy = "BcscBCeID")]
```

Comment:
I used this policy as I think we should be able to use it for different kinds of applications

Reply:
Yes, makes sense